andela/bad_ass-ah-frontend
# 164489904 Allowing a user to fetch his/her bookmarked articles.

**What does this PR do?**

This PR allows a user to get all the articles bookmarked by him. 

**Description of Task to be completed?**

Users should be able to get all bookmarks

**How should this be manually tested?**

After pulling run `npm install` to install all dependencies, then on user profile click the view bookmark button to see all the articles you've bookmarked. 

**What are the relevant pivotal tracker stories?**

[#164489902](https://www.pivotaltracker.com/story/show/164489902)

**Screenshots (if appropriate)**

![image](https://user-images.githubusercontent.com/44478388/59563945-c3469400-9040-11e9-934c-8b5159df0cda.png)
